diff -up sphinx_rtd_theme-3.0.2/setup.cfg.1~ sphinx_rtd_theme-3.0.2/setup.cfg
--- sphinx_rtd_theme-3.0.2/setup.cfg.1~	2024-11-13 12:03:49.327329900 +0100
+++ sphinx_rtd_theme-3.0.2/setup.cfg	2025-09-22 15:09:28.498442567 +0200
@@ -45,7 +45,7 @@ packages = sphinx_rtd_theme
 python_requires = >=3.8
 install_requires = 
 	sphinx >=6,<9
-	docutils >0.18,<0.22
+	docutils >0.18,<0.23
 	sphinxcontrib-jquery >=4,<5
 tests_require = 
 	pytest
